Clarified contribution eligibility and checkins (#1201)
mattermost/mattermost-handbook: contributors/join-us/github-sponsorship.md

[GitHub Sponsors](https://docs.github.com/en/github/supporting-the-open-source-community-with-github-sponsors/about-github-sponsors#about-github-sponsors) allows the developer community to financially support the people and organizations who design, build, and maintain the open source projects they depend on, directly on GitHub.

## Who is eligible?

Mattermost GitHub sponsorship is considered for [MVP](https://developers.mattermost.com/contribute/mvp/)-like contributors \("Most Valued Professional"\) in any of the [available regions](https://github.com/sponsors) supported by GitHub.

MVP-like contributors:

- Focus on contributions that have a wider impact that many can benefit from.
- Have proven success working asynchronously in a self-serve, autonomous way that doesn’t  require many direct touch points with Mattermost teams to develop, test, or document their contribution.
- Give extra effort in a way that exceeds what’s expected.
- Has, shares, and delivers on a clear vision.
- Steps forward to assist when opportunities present themselves.
- Has demonstrated empathy and understanding for the software development process.

## Eligible contribution areas

> - Technical peer help on demand via public forums, including but not limited to: Mattermost Community Server, Mattermost User Forum, and Mattermost sub-Reddit forum.

Reviewer: ```suggestion
- Technical peer help on demand via public forums, including but not limited to: Mattermost Community server, Mattermost User Forum, and the Mattermost subreddit on Reddit.
```

- Product localization via string translation (known also as l10n and i18n).
- Lead monthly product localization delivery (cherry-pick L10N PRs, weekly community communication, data analytics/reporting, develop and deliver nurture campaigns to increase translation quality).
- Product evangelism for technical communities.
- Fix product issues and add new product functionality to improve the user experience.
- Address user journey gaps such as migration tooling.
- Customer-requested product integrations.

## What types of contributions aren't eligible?

> - Code changes to features that Mattermost charges money to use. This includes features available only to Professional or Enterprise customers that aren't open-sourced, and code within the /enterprise GitHub repository where the majority of paid Enterprise features are maintained.

Reviewer: ```suggestion
- Code changes to features that Mattermost charges money to use. This includes features available only to Professional or Enterprise customers that aren't open-sourced, and code within the ``/enterprise`` GitHub repository where the majority of paid Enterprise features are maintained.
```

- Mattermost product and development documentation contributions. Documentation isn’t currently eligible because of the range of effort complexity involved (from typo fixes to new content development) which doesn’t directly map to the [existing reimbursement structure](https://docs.google.com/document/d/1P88K3p-wDbV9nLZNkBnMk_SJvt90gzlRq3Pj0ZPBOZg/edit#).

## Process outline

1. A Mattermost staff member recommends a contributor for sponsorship and writes a short motivation to support their recommendation. This should be addressed to their direct manager as well as the program lead.

> 2. [MLT](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#mlt) senior leader approves the recommendation and notifies the program lead.

Reviewer: ```suggestion
2. An [MLT](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#mlt) senior leader approves the recommendation and notifies the program lead.
```
